cliche/services/tvtropes/crawler.py

```diff
@@ -133,23 +133,20 @@ def fetch_link(url, session, *, log_prefix=''):
         return False, None, None, None, final_url
     tree = document_fromstring(r.text)
     try:
-        namespace = tree.xpath('//div[@class="pagetitle"]')[0] \
-            .text.strip()[:-1]
+        name = (tree.find_class('article_title')[0]).text_content()
```

Comment:
Does this work? I inspected tvtropes, and it seems the structure is like below:
'''
<div class="pagetitle">

                        
                        
                        
                        <div class="article_title"><h1><span>Home Page</span></h1></div>
                        
                    </div>
'''